spectrum/input.py

```diff
@@ -228,6 +228,10 @@ def logout(self):
         profile = logged_in_page.soup.select_one(profile_selector)
         assert profile is None, ("Found %s in %s response\n%s" % (profile_selector, logged_in_page.status_code, logged_in_page.content))
 
+    def check(self, path):
```

Comment:
`path` is imported from a module, so the linter complains that a variable is defined shadowing it